releases/solana/2025-09-upgrade-anchor.md

```diff
@@ -43,11 +43,11 @@ This is the Solana GMP upgrade doc for anchor TO anchor programs.
     cd axelar-amplifier-solana
 
     # Compile the Solana programs
-    solana-verify build --base-image $BASE_IMAGE --library-name axelar_solana_gas_service
-    solana-verify build --base-image $BASE_IMAGE --library-name axelar_solana_gateway
-    solana-verify build --base-image $BASE_IMAGE --library-name axelar_solana_governance
-    solana-verify build --base-image $BASE_IMAGE --library-name axelar_solana_multicall
-    solana-verify build --base-image $BASE_IMAGE --library-name axelar_solana_memo_program
+    solana-verify build --base-image $BASE_IMAGE --library-name solana_axelar_gas_service
```

Comment:
Once release process is merged and we have pre releases/release tags, build details should be removed